src/devices.js

```diff
@@ -254,12 +304,10 @@ function installStep(step) {
                     "Fastboot is scanning for devices"
                   );
                   function fastbootWait() {
-                    return fastboot
-                      .waitForDevice()
-                      .then(resolve)
-                      .catch(() =>
-                        mainEvent.emit("user:connection-lost", fastbootWait)
-                      );
+                    return fastboot.hasAccess().then(access => {
```

Comment:
`fastboot.hasAccess()` uses `fastboot devices` under the hood. The idea is that if the connection was lost because the user accidentally disconnected their device, they'll just plug it back in and then click the button. If the device is still not seen, they just change another thing, click the button again, etc. A little safer this way.